src/matrix/room/BaseRoom.js

```diff
@@ -378,8 +381,44 @@ export class BaseRoom extends EventEmitter {
         return this._summary.data.membership;
     }
 
-    get powerLevels() {
-        return this._timeline.powerLevels;
+    async loadPowerLevels() {
+        const timelineReader = new TimelineReader({
+            roomId: this._roomId,
+            storage: this._storage,
+            fragmentIdComparer: this._fragmentIdComparer
+        });
+        const txn = await this._storage.readTxn(
+            timelineReader.readTxnStores.concat(this._storage.storeNames.roomMembers, this._storage.storeNames.roomState)
```

Comment:
the transaction should only use the `roomState` store, as that is the only store you'll be reading from here.